src/apps/concepts/components/ViewConceptsHeader.tsx

```diff
@@ -37,53 +37,61 @@ const ViewConceptsHeader: React.FC<Props> = ({
         handleSwitchSourceClose
     ] = useAnchor();
 
+    const getTitleBasedOnContainerType = () => {
+        return isAddToDictionary
+            ? `Import existing concept from ${getContainerIdFromUrl(
+                containerUrl
+            )}`
+            : `Concepts in ${
+                containerType === DICTIONARY_VERSION_CONTAINER ? "v" : ""
+            }${getContainerIdFromUrl(containerUrl)}`;
+    };
+
+    const showSwitchSourceBasedOnContainerType = () => {
+        return !isAddToDictionary ? null : (
```

Comment:
it is better if we remove the ternary operator and use a if condition, code will look much cleaner